packages/expo-status-bar/tsconfig.json

```diff
@@ -0,0 +1,21 @@
+
```

Comment:
We used to be using `tsconfig.json` generated from `expo-module-scripts` 🤔

Reply:
hm, another `et cu` thing, will look into it